supplementary_style_guide/glossary_terms_conventions/product_conventions/amq7.adoc

```diff
@@ -604,7 +604,7 @@
 
 *Incorrect forms*:
 
-*See also*: xref:target[target]
+*See also*: xref:target-amq[target]
```

Comment:
I think there is one more instance of "xref:target" in the AMQ file (sender) that needs to be updated to `target-amq`.

Reply:
Done, thankyou.